android/CMakeLists.txt

```diff
@@ -57,70 +35,24 @@ add_library(
   cpp-adapter.cpp
 )
 
-add_definitions(
-  ${SQLITE_FLAGS}
+set_target_properties(
+  ${PACKAGE_NAME} PROPERTIES
+  CXX_STANDARD 17
+  CXX_EXTENSIONS OFF
+  POSITION_INDEPENDENT_CODE ON
 )
 
-# find fbjni package
-file (GLOB LIBFBJNI_INCLUDE_DIR "${BUILD_DIR}/fbjni-*-headers.jar/")
+find_package(ReactAndroid REQUIRED CONFIG)
+find_package(fbjni REQUIRED CONFIG)
+find_library(LOG_LIB log)
 
-target_include_directories(
-        ${PACKAGE_NAME}
-        PRIVATE
-        # --- fbjni ---
-        "${LIBFBJNI_INCLUDE_DIR}"
-        # --- React Native ---
-        "${NODE_MODULES_DIR}/react-native/React"
-        "${NODE_MODULES_DIR}/react-native/React/Base"
-        "${NODE_MODULES_DIR}/react-native/ReactAndroid/src/main/jni"
-        "${NODE_MODULES_DIR}/react-native/ReactAndroid/src/main/java/com/facebook/react/turbomodule/core/jni"
-        "${NODE_MODULES_DIR}/react-native/ReactCommon"
-        "${NODE_MODULES_DIR}/react-native/ReactCommon/callinvoker"
-        "${NODE_MODULES_DIR}/react-native/ReactCommon/jsi"
-        "${NODE_MODULES_DIR}/hermes-engine/android/include/"
-        ${INCLUDE_JSI_CPP} # only on older RN versions
-        ${INCLUDE_JSIDYNAMIC_CPP} # only on older RN versions
-)
-
-file (GLOB LIBRN_DIR "${BUILD_DIR}/react-native-0*/jni/${ANDROID_ABI}")
-
-find_library(
-        FBJNI_LIB
-        fbjni
-        PATHS ${LIBRN_DIR}
-        NO_CMAKE_FIND_ROOT_PATH
-)
-
-find_library(
-        REACT_NATIVE_JNI_LIB
-        reactnativejni
-        PATHS ${LIBRN_DIR}
-        NO_CMAKE_FIND_ROOT_PATH
-)
-if(${REACT_NATIVE_VERSION} LESS 66)
-        # JSI lib didn't exist on RN 0.65 and before. Simply omit it.
-        set (JSI_LIB "")
-else()
-        # RN 0.66 distributes libjsi.so, can be used instead of compiling jsi.cpp manually.
-        find_library(
-                JSI_LIB
-                jsi
-                PATHS ${LIBRN_DIR}
-                NO_CMAKE_FIND_ROOT_PATH
-        )
-endif()
-
-find_library(
-        LOG_LIB
-        log
-)
-
-# target_link_libraries(sequel fbjni::fbjni android log)
 target_link_libraries(
-        ${PACKAGE_NAME}
-        ${LOG_LIB}
-        ${JSI_LIB}
-        ${REACT_NATIVE_JNI_LIB}
-        ${FBJNI_LIB}
-        android
+  ${PACKAGE_NAME}
+  ${LOG_LIB}
+  fbjni::fbjni
+  ReactAndroid::jsi
+  ReactAndroid::turbomodulejsijni
+  ReactAndroid::react_nativemodule_core
+  ReactAndroid::hermes_executor
```

Comment:
@ospfranco this library is awesome, thanks for taking the time! From my initial tests `SELECT`s seem to be around 3-5x faster for 100k records vs all the bridge-based libraries. Bulk insert transactions might even be 8x faster in some cases.

I had an error getting the 8.0.0-beta.2 release working with a JSC-based app:

> Error: Exception in HostFunction: java.lang.UnsatisfiedLinkError: dlopen failed: library "libhermes_executor.so" not found

I borrowed this idea from `expo-modules-core` but think it could be fixed by doing something like:

```
if(${USE_HERMES})
  set(JSEXECUTOR_LIB ReactAndroid::hermes_executor)
else()
  set(JSEXECUTOR_LIB ReactAndroid::jscexecutor)
endif()

target_link_libraries(
  ${PACKAGE_NAME}
  ${LOG_LIB}
  fbjni::fbjni
  ReactAndroid::jsi
  ReactAndroid::turbomodulejsijni
  ReactAndroid::react_nativemodule_core
  ${JSEXECUTOR_LIB}
  android
)
```